fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/TautologicalValue.java

```diff
@@ -59,6 +64,13 @@ public final class TautologicalValue extends AbstractValue implements BooleanVal
     private TautologicalValue() {
     }
 
+    @Nonnull
+    @Override
+    public ExplainTokensWithPrecedence explain(@Nonnull final Iterable<Supplier<ExplainTokensWithPrecedence>> explainSuppliers) {
+        Verify.verify(Iterables.isEmpty(explainSuppliers));
+        return ExplainTokensWithPrecedence.of(new ExplainTokens().addKeyword("TRUE"));
```

Comment:
This can be confused with the boolean constant `TRUE`, but it is ok, considering that we'll eventually get rid of this `Value`.